docs/content/docs/dev/datastream/operators/asyncio.md

```diff
@@ -194,6 +194,8 @@ The following two parameters control the asynchronous operations:
 
 When an async I/O request times out, by default an exception is thrown and job is restarted.
 If you want to handle timeouts, you can override the `AsyncFunction#timeout` method.
+Make sure you call `ResultFuture.complete()` or `ResultFuture.completeExceptionally()` when overriding
```

Comment:
Should we explicitly point out `complete(Collections.emptyList())` in case the user does not want to emit any record?

Reply:
Did not know that. Let me check.

Reply:
Added